apps/api/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "api",
 	"private": true,
 	"scripts": {
-		"dev": "python src/dev.py",
+		"dev": "./run-docker.sh",
```

Comment:
Thought: do we need an entire separate executable file for this, or could we write the statements together here?
```shell
docker compose up --build && docker compose down
```
Could see it being cleaner both ways, unsure which is better.

Reply:
If we have it this way though, I don't think `docker compose down` will run when pressing `ctrl + c`. I tried it on my own machine:

`./run-docker.sh`
<img width="403" alt="image" src="https://github.com/HackAtUCI/irvinehacks-site-2024/assets/28026520/4f09786f-6422-43d9-87be-7817035db3b4">

vs. 

chained commands
<img width="532" alt="image" src="https://github.com/HackAtUCI/irvinehacks-site-2024/assets/28026520/12c1bea9-3d0e-49d4-abcf-3ee909aa900d">

Reply:
Got it, thanks for discovering how to get both up and down working with the file.